src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -472,6 +472,34 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
   ============================================================
+                                      InjectTargetPathMetadata
+  
+  Update TargetPathWithTargetPlatformMoniker with target framework
+  identifier and version metadata.  This is so that the
+  ImplicitlyExpandNETStandardFacades target can determine if a
+  referenced project needs the .NET Standard facades even if
+  the project hasn't been compiled to disk yet.
+  
+  See https://github.com/dotnet/sdk/issues/1403 for more context
+  ============================================================
+  -->
+  
+  <Target Name="InjectTargetPathMetadata"
+        BeforeTargets="GetTargetPath"
+        DependsOnTargets="GetTargetPathWithTargetPlatformMoniker">
```

Comment:
This will return different metadata when the project entry point is `Build` versus `GetTargetPath`, won't it? If building only `GetTargetPath`, the is-any-dependency-NETStandard target won't fire and the return value won't be transitive. But when returned from `Build`, `ResolveReferences` will have run and the return value will be transitive.

The behaviors could be aligned by adding reference resolution to this target's dependencies, but that would be a pretty big change; IDE and other specialized builds assume that `GetTargetPath` is very cheap and it wouldn't be if it was recursive.